1_historical_RS_data_collation/src/combine_metadata_with_pulls.R

```diff
@@ -0,0 +1,163 @@
+#function to grab the row id
```

Comment:
Just a quick note: I've incorporated these functions into the general target function below in the version I'm currently working on.